components/link/Link.js

```diff
@@ -0,0 +1,54 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import s from './theme.css';
+
+class Link extends PureComponent {
+  static propTypes = {
+    children: PropTypes.node,
+    className: PropTypes.string,
+    external: PropTypes.bool,
+    onClick: PropTypes.func,
+    url: PropTypes.string,
+  };
+
+  static defaultProps = {
+    className: '',
+    external: false,
+  };
+
+  render () {
+    const {
+      children,
+      className,
+      external,
+      onClick,
+      url,
+      ...rest
+    } = this.props;
+
+    const classNames = cx(
+      s['link'],
+      className,
+    );
+
+    const target = external ? '_blank' : undefined;
+    const rel = external ? 'noopener noreferrer' : undefined;
```

Comment:
Not really fond of limiting target and rel to these values. I would just not define them as props in this component, and let them sit in the `rest` so you can still pass them to the component, with any value you like.

Reply:
Maybe combine the `target` and `rel` with potential `this.props.target` and `this.props.rel`?